### PR TITLE
example utils documentation added

### PR DESCRIPTION
**Pull request by pseusys**
# Description

Documentations of `dff\utils\testing` was missing, so I added it

# Checklist

- [X] I have covered the code with tests
- [X] I have added comments to my code to help others understand it
- [X] I have updated the documentation to reflect the changes
- [X] I have performed a self-review of the changes

### CODE REVIEW

File: dff/utils/testing/common.py [hunk 1]
```diff
@@ -8,6 +8,12 @@
 
 
 def is_interactive_mode() -> bool:
+    """
+    Checking whether the example code should be run in interactive mode.
+
+    :return: true only if it's being executed by Jupyter kernel AND DISABLE_INTERACTIVE_MODE env variable isn't set.
```
Comment (RLKRo) [inline dff/utils/testing/common.py:14]: I don't see how this is true.

The function returns true if:

1. `IPython` module is not installed
2. shell is not `ZMQInteractiveShell`

It seems the function returns false only if it's being executed by Jupyter kernel.

**Review by RLKRo — CHANGES_REQUESTED** [review]
Also, this branch should have a `docs/` prefix. For reference:
https://gist.github.com/seunggabi/87f8c722d35cd07deb3f649d45a31082

File: dff/utils/testing/common.py (hunk 1 above)
Comment (pseusys) [reply dff/utils/testing/common.py:14]: Generally you are right, it returns false only if it's being executed by Jupyter kernel and environmental variable is not set. That means acually that the example code shouldn't be run in interactive mode.
What's wrong with it?

Comment (RLKRo) [reply dff/utils/testing/common.py:14]: The doc reads `:return: true only if it's being executed by Jupyter kernel`.

**Review by RLKRo — APPROVED** [review]

**Review by kudep — APPROVED** [review]

**Pull request merged**